pkg/service/did/service.go

```diff
@@ -71,16 +74,29 @@ func NewDIDService(config config.DIDServiceConfig, s storage.ServiceStorage, key
 	}
 
 	// instantiate DID resolver
-	resolver, err := did.BuildResolver(config.ResolutionMethods)
+	localResolver, err := did.BuildResolver(config.ResolutionMethods)
 	if err != nil {
 		return nil, errors.Wrap(err, "could not instantiate DID resolver")
 	}
 
 	service := Service{
-		storage:  didStorage,
-		handlers: make(map[didsdk.Method]MethodHandler),
-		keyStore: keyStore,
-		resolver: resolver,
+		storage:          didStorage,
+		handlers:         make(map[didsdk.Method]MethodHandler),
+		keyStore:         keyStore,
+		sdkResolver:      localResolver,
+		methodToResolver: make(map[string]resolve.Resolver),
+	}
+
+	for _, sm := range service.sdkResolver.SupportedMethods() {
+		service.methodToResolver[sm.String()] = service.sdkResolver
+	}
+
+	ur := &resolve.UniversalResolver{
+		Client: http.Client{},
```

Comment:
are there any default params we should set here - like timeouts, etc?

Reply:
this is a good point. Instead of defining defaults, I made the changes so that the context is propagated properly. The timeouts will be inherited from the context. This should also give us spans and traces for free.